smartsim/_core/launcher/colocated.py

```diff
@@ -80,7 +80,6 @@ def write_colocated_launch_script(
 def _build_colocated_wrapper_cmd(
     fs_log: str,
     cpus: int = 1,
-    rai_args: t.Optional[t.Dict[str, str]] = None,
     extra_fs_args: t.Optional[t.Dict[str, str]] = None,
     port: int = 6780,
     ifname: t.Optional[t.Union[str, t.List[str]]] = None,
```

Comment:
Like I mentioned before, I'm not sure how we are planning on doing colocation in the SSv1.0 API, but I'm fairly certain its not going to be through an entrypoint. If you can get it by the CI, consider removing this method entirely!

Reply:
removed method